content/docs/setup/kubernetes/download/index.md

```diff
@@ -51,3 +51,63 @@ To use the Istio release Helm chart repository, add the Istio release repository
 {{< text bash >}}
 $ helm repo add istio.io https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/charts/
 {{< /text >}}
+
+## `istioctl` tab completion
+
+If you are using Bash or ZSH, `istioctl` includes a helpful script that enables tab completion for the currently available `istioctl` commands.
+
+### Installing the `istioctl` tab completion file
+
+If you are using Bash, the `istioctl` tab completion file is located in the `tools` directory. To use it, copy the `istioctl.bash` file to your home directory, then add the following line to source the `istioctl` tab completion file from your `.bashrc` file:
+
+{{< text bash >}}
+$ source ~/istioctl.bash
+{{< /text >}}
+
+For ZSH users, an additional `istioctl` tab completion file is also located in the `tools` directory. In a similar fashion, you can copy the `_istioctl` file to your home directory, then source the `istioctl` tab completion file as follows:
+
+{{< text zsh >}}
+$ source < _istioctl
+{{< /text >}}
+
+{{< tip >}}
+If you get an error like `complete:13: command not found: compdef`, then add the following to the beginning of your `~/.zshrc` file:
+
+{{< text bash >}}
+$ autoload -Uz compinit
+$ compinit
+{{< /text >}}
+
+{{< /tip >}}
+
+You may also add the `_istioctl` file to a directory listed in the `FPATH` variable. To achieve this, place the `_istioctl` file in an existing directory in the `FPATH`, or create a new directory and add it to the `FPATH` variable in your `~/.zshrc` file.
+
+### Prerequisites for macOS
```

Comment:
Prereqs for Linux:

For Linux, to install bash completion, one would use `apt-get install bash-completion` or `yum install bash-completion` for the two common occurances.

For Mac, you already have the instructions.

For Zsh, you could leave off how to add bash completion.

Reply:
@ericvn looks like this PR was recently merged, so I can address this feedback in a separate PR.

Reply:
@ericvn PR to address the above comment is here:
-----> https://github.com/istio/istio.io/pull/4164